buildgroupai/django-davinci-crawling
# Add proxy mesh

**Changes**
* Add proxy mesh to chrome driver on selenium;
* Add proxy mesh to fetch_file to download files;

**Bugs Fixed**
* None

**Tests**
* Add tests for random proxies select on proxy mesh implementation;
* Add a test that fetches a web page using selenium and check if the IP is changing;
* Add a test that downloads a file that returns an IP address and check if the IP is changing.
